### DISCUSSION TITLE
Visual connection between parent issue and a sub-task

### DISCUSSION THREAD

**Discussion by tamalexmak** [community/community#5998] [Projects and Issues]
There is no visual connection between parent and child (sub-task) issue both in board and table view. 

In the current boards one can create a note with the list of sub-task issues, then they will be displayed connected.
![Board_sub-task](https://user-images.githubusercontent.com/68220826/136032574-ee0c0d7d-624f-4dd5-9bbb-17b3afc73928.png)
Would be nice to have similar display of parent issue and sub-tasks in the beta boards.

**Comment by NickTP24** [comment]
Agreed. This would be highly beneficial.
